packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -360,7 +360,7 @@ class TypeMatcher<T> {
 ///    particular configuration and ambient state.
 abstract class Widget {
   /// Initializes [key] for subclasses.
-  const Widget({ this.key });
+  const Widget({ this.key, this.debugTraceData });
```

Comment:
who's going to populate this? This constructor is abstract and none of the subclasses pass it in...